Source/WebKit/UIProcess/API/wpe/WebKitWebViewWPE.cpp

```diff
@@ -30,9 +30,18 @@
 #include <wpe/wpe-platform.h>
 #endif
 
-gboolean webkitWebViewAuthenticate(WebKitWebView*, WebKitAuthenticationRequest*)
+gboolean webkitWebViewAuthenticate(WebKitWebView*, WebKitAuthenticationRequest* request)
 {
-    return FALSE;
+    switch (webkit_authentication_request_get_scheme(request)) {
+    case WEBKIT_AUTHENTICATION_SCHEME_CLIENT_CERTIFICATE_REQUESTED:
```

Comment:
I guess we should handle pin here too

Reply:
The pin case would only happen if a certificate is provided, but not the corresponding password.
I assumed the correct behavior would be to cancel.